pkgs/tools/filesystems/nullfs/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchFromGitHub, kernel }:
+stdenv.mkDerivation rec {
+  pname = "nullfs";
+  version = "0.3-git";
```

Comment:
```suggestion
  version = "0.3";
```

Reply:
0.3 is a different commit hash ...

Reply:
Then we need to name it unstable-XXXX-XX-XX where XXXX-XX-XX is the date of the commit.

Reply:
using v3.0 - new commits only change readme